stob/module.c

```diff
@@ -24,29 +24,29 @@
 #include "module/instance.h"
 
 #if 0 /* unused code */
-static int level_stob_enter(struct m0_module *module);
-static void level_stob_leave(struct m0_module *module);
+M0_UNUSED static int level_stob_enter(struct m0_module *module);
+M0_UNUSED static void level_stob_leave(struct m0_module *module);
 
-static const struct m0_modlev levels_stob[] = {
+M0_UNUSED static const struct m0_modlev levels_stob[] = {
 	[M0_LEVEL_STOB] = {
 		.ml_name = "stob is initialised",
 		.ml_enter = level_stob_enter,
 		.ml_leave = level_stob_leave,
 	}
 };
 
-static int level_stob_enter(struct m0_module *module)
+M0_UNUSED static int level_stob_enter(struct m0_module *module)
 {
 	return m0_stob_types_init();
 }
 
-static void level_stob_leave(struct m0_module *module)
+M0_UNUSED static void level_stob_leave(struct m0_module *module)
 {
 	m0_stob_types_fini();
 }
 #endif
 
-M0_INTERNAL struct m0_stob_module *m0_stob_module__get(void)
+M0_UNUSED M0_INTERNAL struct m0_stob_module *m0_stob_module__get(void)
```

Comment:
Why is this change needed?

Reply:
M0_UNUSED macro is used to mark the  function as unused without commenting  and to tame the compiler  from  warning as unused function

Reply:
But this function is used. For example, at https://github.com/Seagate/cortx-motr/blob/f3e806f684e57e96f39b20ebe31648a8815b1631/stob/type.c#L91.

Reply:
Fixed, that was left over